packages/office-addin-project/src/convert.ts

```diff
@@ -51,15 +51,18 @@ async function asksForUserConfirmation(): Promise<boolean> {
 
 async function backupProject(backupPath: string) {
   const zip: AdmZip = new AdmZip();
-  const outputPath = path.resolve(backupPath);
+  const outputPath: string = path.resolve(backupPath);
+  const rootDir: string = path.resolve(); 
 
-  const files: string[] = fs.readdirSync(__dirname);
+  const files: string[] = fs.readdirSync(rootDir);
   files.forEach((entry) => {
-    const entryStats = fs.lstatSync(entry);
+    const fullPath = path.resolve(entry)
+    const entryStats = fs.lstatSync(fullPath);
+
     if (skipBackup.includes(entry)) {
       // Don't add it to the backup
     } else if (entryStats.isDirectory()) {
-      zip.addLocalFolder(entry);
+      zip.addLocalFolder(entry, entry);
```

Comment:
Can you put a comment on that to explain the need for two entries? (What each entry is doing).

Reply:
That would be part of the adm-zip documentation.  The first argument is the path to pull the file from (in this case relative to the project), and the second argument is the path in the zip file to put it.  I don't know that we want to get into the habit of redocumenting library apis locally . . . it could get out of date and we'd have tons of extra comments all over out code.